<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1081</width>
    <height>706</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Caso de Uso da Biblioteca QTheory (Teoria Das Filas)</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="grpRegistroChegada">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>521</width>
      <height>131</height>
     </rect>
    </property>
    <property name="title">
     <string>Registro das horas de chegadas de clientes</string>
    </property>
    <widget class="QLineEdit" name="txtArquivoChegadas">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>381</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="btnChegadas">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>50</y>
       <width>80</width>
       <height>64</height>
      </rect>
     </property>
     <property name="text">
      <string> Abrir</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>icons/abrir.png</normaloff>icons/abrir.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>22</width>
       <height>22</height>
      </size>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>34</y>
       <width>54</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Arquivo:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtPontosChegadas">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>90</y>
       <width>161</width>
       <height>25</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Linhas do arquivo *.csv onde se inicia cada observação.</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>74</y>
       <width>161</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Pontos (separados por &quot;;&quot; ):</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>74</y>
       <width>161</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Coluna de leitura:</string>
     </property>
    </widget>
    <widget class="QComboBox" name="cbColunasChegadas">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>90</y>
       <width>211</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="grpRegistroChegada_2">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>10</y>
      <width>521</width>
      <height>131</height>
     </rect>
    </property>
    <property name="title">
     <string>Registro das horas de partidas de clientes</string>
    </property>
    <widget class="QLineEdit" name="txtArquivoPartidas">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>381</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="btnPartidas">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>50</y>
       <width>80</width>
       <height>64</height>
      </rect>
     </property>
     <property name="text">
      <string> Abrir</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>icons/abrir.png</normaloff>icons/abrir.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>22</width>
       <height>22</height>
      </size>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>34</y>
       <width>54</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Arquivo:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtPontosPartidas">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>90</y>
       <width>161</width>
       <height>25</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Linhas do arquivo *.csv onde se inicia cada observação.</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>74</y>
       <width>161</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Pontos (separados por &quot;;&quot; ):</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>74</y>
       <width>161</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Coluna de leitura:</string>
     </property>
    </widget>
    <widget class="QComboBox" name="cbColunasPartidas">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>90</y>
       <width>211</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QTableView" name="tbChegadas">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>180</y>
      <width>521</width>
      <height>173</height>
     </rect>
    </property>
    <property name="sizeAdjustPolicy">
     <enum>QAbstractScrollArea::AdjustToContents</enum>
    </property>
    <property name="gridStyle">
     <enum>Qt::DashLine</enum>
    </property>
    <attribute name="horizontalHeaderCascadingSectionResizes">
     <bool>true</bool>
    </attribute>
    <attribute name="horizontalHeaderHighlightSections">
     <bool>false</bool>
    </attribute>
    <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
     <bool>false</bool>
    </attribute>
    <attribute name="horizontalHeaderStretchLastSection">
     <bool>true</bool>
    </attribute>
   </widget>
   <widget class="QComboBox" name="cbModelos">
    <property name="geometry">
     <rect>
      <x>102</x>
      <y>148</y>
      <width>291</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lblModeloTeorico1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>151</y>
      <width>91</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Modelo teórico:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnExecChegadas">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>146</y>
      <width>121</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Executar</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>icons/thunder.png</normaloff>icons/thunder.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>35</width>
      <height>35</height>
     </size>
    </property>
   </widget>
   <widget class="QComboBox" name="cbModelosP">
    <property name="geometry">
     <rect>
      <x>632</x>
      <y>148</y>
      <width>291</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QTableView" name="tbPartidas">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>180</y>
      <width>521</width>
      <height>173</height>
     </rect>
    </property>
    <property name="sizeAdjustPolicy">
     <enum>QAbstractScrollArea::AdjustToContents</enum>
    </property>
    <property name="gridStyle">
     <enum>Qt::DashLine</enum>
    </property>
    <attribute name="horizontalHeaderCascadingSectionResizes">
     <bool>true</bool>
    </attribute>
    <attribute name="horizontalHeaderHighlightSections">
     <bool>false</bool>
    </attribute>
    <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
     <bool>false</bool>
    </attribute>
    <attribute name="horizontalHeaderStretchLastSection">
     <bool>true</bool>
    </attribute>
   </widget>
   <widget class="QLabel" name="lblModeloTeorico1_2">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>151</y>
      <width>91</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Modelo teórico:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnExecPartidas">
    <property name="geometry">
     <rect>
      <x>940</x>
      <y>146</y>
      <width>121</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Executar</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>icons/thunder.png</normaloff>icons/thunder.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>35</width>
      <height>35</height>
     </size>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>360</y>
      <width>341</width>
      <height>331</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="title">
     <string>Distribuição de probabilidade das chegadas</string>
    </property>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>320</width>
       <height>291</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);
border-color: rgb(46, 52, 54);
</string>
     </property>
     <property name="text">
      <string>Graph View</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QWidget" name="horizontalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>301</width>
       <height>271</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout"/>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>360</y>
      <width>341</width>
      <height>331</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="title">
     <string>Distribuição de probabilidade das partidas</string>
    </property>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>320</width>
       <height>291</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);
border-color: rgb(46, 52, 54);
</string>
     </property>
     <property name="text">
      <string>Graph View</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QWidget" name="horizontalLayoutWidget_4">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>301</width>
       <height>271</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout2"/>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>360</y>
      <width>351</width>
      <height>331</height>
     </rect>
    </property>
    <property name="title">
     <string>Parâmetros da fila</string>
    </property>
    <widget class="QTreeWidget" name="arvore">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>331</width>
       <height>251</height>
      </rect>
     </property>
     <column>
      <property name="text">
       <string notr="true">1</string>
      </property>
     </column>
    </widget>
    <widget class="QPushButton" name="btnExecParametros">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>30</y>
       <width>71</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>icons/thunder.png</normaloff>icons/thunder.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>35</width>
       <height>35</height>
      </size>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>36</y>
       <width>71</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>&quot;n&quot; máximo:</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="spNMax">
     <property name="geometry">
      <rect>
       <x>86</x>
       <y>30</y>
       <width>51</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>143</x>
       <y>36</y>
       <width>61</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Servidores:</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="spNServidores">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>30</y>
       <width>51</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
